.gitignore

```diff
@@ -2,6 +2,7 @@
 test.db
 .coverage
 .pytest_cache/
+.python-version
```

Comment:
This is now duplicating what is in the default branch:
https://github.com/encode/starlette/blob/38185f39de3f92b2781826b6e0d07ca52b6c9b77/.gitignore#L11